README.md

```diff
@@ -1,6 +1,44 @@
-Datashader
-----------
+# Datashader
 
 [![Build Status](https://travis-ci.org/bokeh/datashader.svg)](https://travis-ci.org/bokeh/datashader)
 
-Per-pixel aggregation and shading system. Work in progress.
+The Datashader project is focused on building better ways to visualize
+very large datasets, using intelligent server-side downsampling,
+automatic computations performed on the data as it progresses through
+the visualization pipeline, and other related techniques.  The project
+is under active development, and all the code and documentation is
+subject to frequent changes.
+
+## Getting Started
+
+```
+# grab the master branch of the datashader repo
+git clone https://github.com/bokeh/datashader.git
+
+# Create a new conda environment
+conda create -n datashader python=2.7
+source activate datashader
+
+# Install required packages
+conda install pillow numba dynd-python pandas datashape
+
+# Install the datashader library
+cd datashader
+python setup.py develop
+
+# Install Bokeh for running examples
+conda install -c https://conda.anaconda.org/bokeh/channel/dev bokeh
+```
+
+**Run the examples**
+
+Currently requires taxi.castra/, which must be obtained separately, to be
+placed into examples/data/.
```

Comment:
You can now download the data by running the `examples/download_sample_data.py` script. It doesn't have to be run from inside the examples directory, but that's probably the easiest way to explain it.

```
cd examples
# Download the sample data. This may take a while.
python download_sample_data.py
# Run the only example
python dashboard.py
```

Perhaps the discussion of the examples should be moved to a second readme inside the examples folder. There you can enumerate the different examples (when there are more), and also discuss how to download the data.